build_scripts/release_scripts/docker-build.sh

```diff
@@ -1,134 +0,0 @@
-#!/bin/bash
```

Comment:
You have to add a small script in the root with instructions on how to start building. It is cumbersome to look into the Dockerfile to see what parameters are supported. I had to extract the build command from the Jenkins file

Reply:
I would rather adjust the documentation / add a comment to the README? @mikaelronstrom, what's your opinion?